yarn.lock

```diff
@@ -17162,9 +17162,9 @@ cyclist@^1.0.1:
   resolved "https://registry.yarnpkg.com/cyclist/-/cyclist-1.0.1.tgz#596e9698fd0c80e12038c2b82d6eb1b35b6224d9"
   integrity sha1-WW6WmP0MgOEgOMK4LW6xs1tiJNk=
 
-cypress-example-kitchensink@cypress-io/cypress-example-kitchensink#feat/use-supportFiles:
+cypress-example-kitchensink@cypress-io/cypress-example-kitchensink#8389bb6b09c59c2720ceeaadecc8eaa2d4d6551d:
```

Comment:
I don't think this is necessary anymore. 10.0-release and my branch are now passing without it.

Reply:
The `packages/example` build scripts are not compatible with Windows, so using the pre-renamed files available in this commit actually are necessary for Windows.